pulp_container/app/settings.py

```diff
@@ -1,4 +1,12 @@
+from copy import deepcopy
+from django.conf import settings
```

Comment:
I don' t recommend importing `django.conf.settings` here, you can use dynaconf inline merging with `@ ` and  `__` (as in my other comment)

Or you can add a `pulp_container/app/dynaconf_hooks.py` and then do any conditionals or updates there.

`dynaconf_hooks.py`
```py
def post(settings):
    data = {"dynaconf_merge": True}
    data["REST_FRAMEWORK"]["EXCEPTION_HANDLER"] = "..."
    return data
```

The reason is that as settings is formed by the merging of multiple sources the final state is not the one you get grom django.conf at this point here.